docs/current_thinking.rst

```diff
@@ -0,0 +1,18 @@
+.. _libCellMLCurrentThinking:
+
+==============================
+Current Thinking for libCellML
+==============================
+
+This document simply outlines some of the rationale that has an influence on how the codebase is developed.
+
+ - Not dealing with external documents
+ 
+  - absolves libcellml from fetching files and communicating across the internet.
```

Comment:
Nitpicking: no full stop (and thus be consistent with the rest of the list). I imagine that this bullet item and the two following ones are sub-items of "Not dealing with external documents"? In any case, why not make them start with a capital letter?